src/io/flutter/utils/HtmlBuilder.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.utils;
+
+public class HtmlBuilder {
+
+  public static String span(String style, String contents) {
+    return "<span style = \"" + style + "\"> " + contents + "</span>";
+  }
+
+  public static String span(String contents) {
+    return tag("span", contents);
+  }
+
+  public static String attrs(String attribute, String value) {
+    return attribute + " = \"" + value + "\"";
+  }
+
+  public static String html(String... contents) {
+    return tag("html", join(contents));
+  }
+
+  public static String pre(String... contents) {
+    return tag("pre", join(contents));
+  }
+
+  public static String body(String... contents) {
+    return join(contents);
+  }
+
+  private static String join(String... contents) {
+    final StringBuilder sb = new StringBuilder();
+    for (String c : contents) {
+      sb.append(c);
+      sb.append(java.lang.System.lineSeparator());
```

Comment:
surprising the line separator has any impact given this is html.
Also minor nit: Don't we always want just '\n' regardless of the platform?